README.md

```diff
@@ -1,3 +1,8 @@
+
+
+[![Python tests](https://github.com/cvnlab/GLMsingle/actions/workflows/run_tests_python.yml/badge.svg)](https://github.com/cvnlab/GLMsingle/actions/workflows/run_tests_python.yml)
+[![Python demos](https://github.com/cvnlab/GLMsingle/actions/workflows/run_demos_python.yml/badge.svg)](https://github.com/cvnlab/GLMsingle/actions/workflows/run_demos_python.yml)
+[![Python-coverage](https://codecov.io/gh/Remi-Gau/GLMsingle/branch/main/graph/badge.svg?token=H75TAAUVSW)](https://codecov.io/gh/Remi-Gau/GLMsingle)
```

Comment:
The code coverage is not yet uploaded and so this might require some tweaking after merging:
- setting up an account for cvnlab on codecov (can simply link the github account)
- changing the URL of this badge

But once set up you can know how pull request change the code coverage: like this
https://github.com/Remi-Gau/GLMsingle/pull/3#issuecomment-1096863979